README.wget-at.md

```diff
@@ -30,197 +39,15 @@ after which the project can be run:
-        poppler-utils
-
-Version 1.4.4 of Zstd is required, so this needs to be built from source for compatibility with arguments provided from other Alpine builds:
-
-    # Source: https://git.alpinelinux.org/aports/tree/main/zstd/APKBUILD?h=3.15-stable
-    git clone https://github.com/facebook/zstd.git --depth 1 --branch v1.4.4
-    cd zstd && export CFLAGS="-O2" && \
-        make -C lib HAVE_PTHREAD=1 HAVE_ZLIB=0 HAVE_LZMA=0 HAVE_LZ4=0 lib-mt && \
-        make -C programs HAVE_PTHREAD=1 HAVE_ZLIB=0 HAVE_LZMA=0 HAVE_LZ4=0 && \
-        make -C contrib/pzstd && \
-        make PREFIX="/usr" install && \
-        cd ..
-
-Close the repo and build Wget-AT:
-
-    git clone https://github.com/ArchiveTeam/{{REPO_NAME}}
-    cd {{REPO_NAME}}; ./get-wget-lua.sh
-
-Run the project with
-
-    # uncomments the next line if you want to use a virtualenv (sh/bash example)
-    #python3 -m venv --prompt at .venv && source .venv/bin/activate
-    pip install --upgrade pip setuptools wheel
-    pip install --upgrade seesaw zstandard requests
-    run-pipeline3 pipeline.py --concurrent 2 --address '127.0.0.1' YOURNICKHERE
-
-### For FreeBSD:
-
-Honestly, I have no idea. `./get-wget-lua.sh` supposedly doesn't work due to differences in the `tar` that ships with FreeBSD. Another problem is the apparent absence of Lua 5.1 development headers. If you figure this out, please do let us know on IRC (irc.hackint.org #archiveteam).
-
-Troubleshooting
-=========================
-
-Broken? These are some of the possible solutions:
-
-### wget-lua was not successfully built
-
-If you get errors about `wget.pod` or something similar, the documentation failed to compile - wget-lua, however, compiled fine. Try this:
-
-    cd get-wget-lua.tmp
-    mv src/wget ../wget-lua
-    cd ..
-
-The `get-wget-lua.tmp` name may be inaccurate. If you have a folder with a similar but different name, use that instead and please let us know on IRC what folder name you had!
-
-Optionally, if you know what you're doing, you may want to use wgetpod.patch.
-
-### Problem with gnutls or openssl during get-wget-lua
-
-Please ensure that gnutls-dev(el) and openssl-dev(el) are installed.
-
-### ImportError: No module named seesaw
-
-If you're sure that you followed the steps to install `seesaw`, permissions on your module directory may be set incorrectly. Try the following:
-
-    chmod o+rX -R /usr/local/lib/python2.7/dist-packages
-
-### run-pipeline: command not found
-
-Install `seesaw` using `pip2` instead of `pip`.
+Be sure to replace `YOURNICKHERE` with the nickname that you want to be shown as on the tracker. You don't need to register it, just pick a nickname you like.
 
-    pip2 install seesaw
+At this point, there will be a web interface running at http://localhost:8001/. If you don't know or care what this is, you can just ignore it—otherwise, it gives you a fancy view of what's going on.
```

Comment:
This doesn't happen in most Docker containers, IIRC.

Reply:
Yeah it's usually disabled by default using command line arguments